egs/librispeech/ASR/conv_emformer_transducer_stateless/emformer.py

```diff
@@ -1580,10 +1584,8 @@ def infer(
         chunk_mask = make_pad_mask(output_lengths).to(x.device)
         memory_mask = (
             (
-                torch.div(
-                    num_processed_frames,
-                    self.chunk_length,
-                    rounding_mode="floor",
+                (
+                    num_processed_frames << int(math.log(self.chunk_length, 2))
```

Comment:
Please cache the result of  `int(math.log(self.chunk_length, 2))`
in the constructor.